src/OpenLoco/src/Windows/Cheats.cpp

```diff
@@ -643,7 +643,7 @@ namespace OpenLoco::Ui::Windows::Cheats
 
         static void onMouseDown(Window& self, WidgetIndex_t widgetIndex)
         {
-            if (widgetIndex == Widx::target_company_dropdown)
+            if (widgetIndex == Widx::target_company_dropdown_btn)
                 Dropdown::populateCompanySelect(&self, &self.widgets[widgetIndex]);
```

Comment:
You forgot to update the second line. From https://github.com/OpenLoco/OpenLoco/pull/2635#discussion_r1767549803:
```diff
         static void onMouseDown(Window& self, WidgetIndex_t widgetIndex)
         {
-            if (widgetIndex == Widx::target_company_dropdown)
-                Dropdown::populateCompanySelect(&self, &self.widgets[widgetIndex]);
+            if (widgetIndex == Widx::target_company_dropdown_btn)
+                Dropdown::populateCompanySelect(&self, &self.widgets[widgetIndex - 1]);
```

Right now, it looks like this:
![Screenshot (3)](https://github.com/user-attachments/assets/607a8fda-40e1-4554-8745-b91b7ab8186d)

Reply:
Changed.